presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestAnalyzer.java

```diff
@@ -266,6 +266,7 @@ public void testInvalidAttribute()
         assertFails(MISSING_ATTRIBUTE, "SELECT * FROM t1 ORDER BY f");
         assertFails(MISSING_ATTRIBUTE, "SELECT count(*) FROM t1 GROUP BY f");
         assertFails(MISSING_ATTRIBUTE, "SELECT * FROM t1 WHERE f > 1");
+        assertFails(MISSING_ATTRIBUTE, "SELECT t.y FROM (VALUES 1) t(x)");
```

Comment:
You are right. It does not check the right error message. I'll make it a separate test and check its exact error message. Thanks!